src/aot/aot.cpp

```diff
@@ -99,6 +99,72 @@ int load_bytecode(BPFtrace &bpftrace, uint8_t *ptr, size_t len)
   return 0;
 }
 
+// Generates contents of the .BTAOT ELF section.
+std::optional<std::vector<uint8_t>> generate_section(
```

Comment:
naming nit: maybe 'generate_BTAOT_section' (then you could maybe ditch the comment)